src/System.Security.Cryptography.Cng/src/System/Security/Cryptography/RSACng.ImportExport.cs

```diff
@@ -38,10 +38,13 @@ public override void ImportParameters(RSAParameters parameters)
             unsafe
             {
                 if (parameters.Exponent == null || parameters.Modulus == null)
-                    throw new ArgumentException(SR.Cryptography_InvalidRsaParameters);
+                    throw new CryptographicException(SR.Cryptography_InvalidRsaParameters);
 
                 bool includePrivate = parameters.P != null && parameters.Q != null;
 
+                if (includePrivate && parameters.DP == null)
```

Comment:
I was trying not to change 4.6 behavior any more than necessary, but I suspect we’re losing that battle anyway…